cli/command/cli.go

```diff
@@ -281,6 +281,11 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions, ops ...CLIOption)
 	}
 	filterResourceAttributesEnvvar()
 
+	meta, err := cli.contextStore.GetMetadata(cli.currentContext)
```

Comment:
Perhaps we should short-circuit;

- if `cli.currentContext` is `DefaultContextName`
- if `os.Getenv("GODEBUG") != ""`

Because in both cases, we won't be overriding `GODEBUG` currently, so we wouldn't have to resolve the context and read if from disk

Reply:
sure, I'll add an if check quick